cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -18,7 +18,7 @@
 gtl_params = {
     'Cloud Providers': CloudProvider,
     'Availability Zones': AvailabilityZone,
-    'Flavors': Flavor,
+    'Flavors': FlavorCollection,
```

Comment:
Navigating directly with with classes, especially collection registered with `All` is not recommended; should create a object with passing the current appliance and then navigate(Line 42)